tests/backup/backup_test.go

```
@@ -6295,12 +6298,24 @@ var _ = Describe("{DeleteAllBackupObjects}", func() {
 			log.InfoD("Taking backup of applications")
 			ctx, err := backup.GetAdminCtxFromSecret()
 			dash.VerifyFatal(err, nil, "Getting context")
-			preRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[0])
-			postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[0])
+			if len(preRuleNameList) > 0 {
+				preRuleUid, _ = Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[0])
+				preRuleName = preRuleNameList[0]
+			} else {
+				preRuleUid = ""
+				preRuleName = ""
+			}
+			if len(postRuleNameList) > 0 {
+				postRuleUid, _ = Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[0])
```

why errors are ignored here?

---

Done